### DISCUSSION TITLE
Locking the window size in exported standalone apps

### DISCUSSION THREAD

**Discussion by damiensellier** [RomanKubiak/ctrlr#370] [Q&A]
Hi everybody, we talked a lot about being able to resize the canvas for exported VSTs and AU here with solutions. 
I was wondering if it was possible to lock permanently the size of the window for exported standalone pannels?
Maybe there's a way to set the Bottom*Right Corner, non draggable?

Right now I can drag the corner and it reveals the striped canevas.

Thanks!

![resizeable standalone](https://user-images.githubusercontent.com/55164385/137642843-e52abb7d-a4fc-45dc-8657-b0e33df58bb1.jpg)

**Comment by Godlike-Productions** [comment]
Use one of the latest versions of ctrlr that had the window size locked would be my suggestion.  Not sure if there is a way to lock it programatically.

**Reply by damiensellier** [reply]
Hi Jason, yes you are right let's take advantage from this issue. Can you please remind me what exact version it is?

**Reply by Godlike-Productions** [reply]
I think it was 5.3.201

**Reply by damiensellier** [reply]
Thanks mate! I'll check that next week and confirm here.
